tests/unit/converters/test_tabular_converter.py

```diff
@@ -980,9 +980,9 @@ def test_lookup_ids(converter: TabularConverter):
         reference,
         pd.DataFrame(
             [
-                ["foo", None, 1, 2, None],
-                ["node", None, 1, None, 2],
-                ["node", "bar", 1, 2, None],
+                ["foo", np.nan, 1, 2, np.nan],
+                ["node", np.nan, 1, np.nan, 2],
+                ["node", "bar", 1, 2, np.nan],
```

Comment:
Is this change from `None` to `np.nan` due to change in pgm? Perhaps we could use another pair of eyes to confirm this because I don't have this knowledge without digging deep. (@petersalemink95 )

Reply:
no this is needed due to deprecated behaviour in `pandas` (according to `Mismatched null-like values nan and None found.`)